17 primos/listAllPrime.java

```diff
@@ -0,0 +1,39 @@
+/*
+ listing out all prime number from a number m
+ ex input =30
+ output:
+ 2
+ 3
+ 5
+ 7
+ 11
+ 13
+ 17
+ 19
+ 23
+ 29
+ */
+class prime{
+  int i;
+  prime(int a){
```

Comment:
Good, thank you!  What is the reason to put the code in the constructor? What do you think about make a static method in the class listAllPrime? Also remember java standar, the class name, must start with a capital letter.

Reply:
My bad i started learning java this semester and yeah i put the code in the constructor because it looks good.

Reply:
Constructor is for initialize variables or call class methods. Your code works, but my question is about software design. I glad to help you to improve your programming skils, I recomend read the book clean code.